images/linux/scripts/installers/heroku.sh

```diff
@@ -4,7 +4,13 @@
 ##  Desc:  Installs Heroku CLI
 ################################################################################
 
+REPO_URL="https://cli-assets.heroku.com/apt"
+
 # Install Heroku CLI
 curl https://cli-assets.heroku.com/install-ubuntu.sh | sh
```

Comment:
@vsafonkin looks like we can install it without apt-get at all https://devcenter.heroku.com/articles/heroku-cli#standalone-installation
https://cli-assets.heroku.com/install.sh